rebar.config

```
@@ -3,22 +3,15 @@
 
 {erl_opts, [
 	debug_info,
-	{platform_define, "^(R|1|(20)|(21)|(22.0))", old_hmac_api}
+	{platform_define, "^(R|1|(20)|(21)|(22.0))", old_hmac_api},
+	warnings_as_errors
 ]}.
 
-%% Require at least R14B03. (couchdb requires this version or newer, and this code hasn't been tested on earlier)
-{require_min_otp_vsn, "R14B03"}.
+{cover_enabled, true}.
 
-%% Ensure the ebin directory exists before we try to put files there.
-{pre_hooks, [
-	{compile, "mkdir -p ebin"}
-]}.
-
-
-%% == xref ==
+{eunit_opts, [verbose]}.
 
-%% Enable xref warnings.
-{xref_warnings, true}.
+{plugins, [{eqc_rebar, {git, "https://github.com/Quviq/eqc-rebar", {branch, "master"}}}]}.
 
-%% xref checks to run
-{xref_checks, [undefined_function_calls]}.
+{xref_checks,[undefined_function_calls,undefined_functions,locals_not_used,
+    deprecated_function_calls, deprecated_functions]}.
```

We can do without `undefined_functions` and `deprecated_functions`.
```suggestion
{xref_checks,[undefined_function_calls,locals_not_used,
    deprecated_function_calls]}.
```